pkg/commands/hosting_service/definitions.go

```diff
@@ -24,6 +25,7 @@ var bitbucketServiceDef = ServiceDefinition{
 	pullRequestURLIntoDefaultBranch: "/pull-requests/new?source={{.From}}&t=1",
 	pullRequestURLIntoTargetBranch:  "/pull-requests/new?source={{.From}}&dest={{.To}}&t=1",
 	commitURL:                       "/commits/{{.CommitSha}}",
+	branchURL:                       "/branch/{{.BranchName}}",
```

Comment:
This isn't the same as `github's` tree view of a certain branch, but I'm not sure what would people prefer. I was expecting for it to open the view of the entire repo, but on a different branch.

This opens up a view of the actual branch, shows the most recent commits.
![image](https://user-images.githubusercontent.com/8365241/181876741-1778c5a5-c703-4d15-8b5f-bc00e8adfa18.png)

There is a button to take the user to view the source of the current branch, and the URL pattern looks like this:
`/src/<SHA of HEAD>/?at=name-of-the-branch`